tests/unit/src/test/scala/tests/DocumentSymbolSuite.scala

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import scala.meta.internal.metals.Buffers
+import scala.meta.internal.metals.DocumentSymbolProvider
+import scala.meta.internal.mtags.Semanticdbs
+import scala.meta.internal.{semanticdb => s}
+import org.eclipse.lsp4j.DocumentSymbol
+import MetalsTestEnrichments._
+
+/**
+ * Checks the positions of document symbols inside a document
+ */
+object DocumentSymbolSuite extends DirectoryExpectSuite("documentSymbol") {
+  val documentSymbolProvider = new DocumentSymbolProvider(new Buffers())
+
+  override def testCases(): List[ExpectTestCase] = {
+    input.scalaFiles.map { file =>
+      ExpectTestCase(
+        file, { () =>
+          val documentSymbols =
+            documentSymbolProvider.documentSymbols(file.file)
+
+          def toTextDocument(
```

Comment:
Is this method needed?

Reply:
It doesn't have to be a method, I'll refactor it into a simple val. (it was more complicated before)